crates/sui-framework/packages/sui-system/sources/sui_system.move

```diff
@@ -641,10 +648,9 @@ module sui_system::sui_system {
         sui_system_state_inner::get_reporters_of(self, addr)
     }
 
-    #[test_only]
     /// Return the current validator set
-    public fun validators(wrapper: &mut SuiSystemState): &ValidatorSet {
-        let self = load_system_state(wrapper);
+    public fun validators(wrapper: &SuiSystemState): &ValidatorSet {
```

Comment:
This will expose ValidatorSet and make it impossible to upgrade

Reply:
I will revert this and return the stake amount instead